packages/ui/src/pages/Consent/index.module.scss

```diff
@@ -6,3 +6,7 @@
   @include _.image-align-center;
   margin-bottom: _.unit(4);
 }
+
+.loadingHolder {
```

Comment:
wrapper? we can align the name of this type of divs

Reply:
updated